modules/javafx.graphics/src/main/java/javafx/css/Stylesheet.java

```diff
@@ -131,9 +131,9 @@ public void setOrigin(StyleOrigin origin) {
     }
 
     /**
-     * Constructs a Stylesheet using the given URL as the base URI. The
+     * Constructs a Stylesheet interpreting the given String as the base URI. The
```

Comment:
This is not public API and the constructor comment about `null` input is not exactly right. I suggest to not touch this at all until changes to the actual code of the class are made.